samples/05_content_publishers/create_and_load_knowledge_graph_backups.ipynb

```diff
@@ -54,6 +54,7 @@
     "Create a backup of an ArcGIS Knowledge graph you have created to store your data model and data. The backup will be in the format of json files that can read to recreate your graph or be shared for others to be able to create the same graph on a different server.\n",
```

Comment:
<p>Everything looks great when built locally.  The word <em>expected </em>in the comments in the 2 cells above renders behind a cell label (Input) - but this seems very minor. I'll merge this and if in production you want it changed, let us know or just break those comments into 2 lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>